Classes/Domain/Model/Changes/AbstractStructuralChange.php

```diff
@@ -178,6 +178,14 @@ protected function finish(NodeInterface $node)
                 $this->getParentDomAddress()->getFusionPath() &&
                 $this->getParentDomAddress()->getContextPath() === $node->getParent()->getContextPath()
             ) {
+                // Include nested nodedata for with the rendered content as the change usually only contains the updated node
+                if ($node->hasChildNodes()) {
```

Comment:
why dont we modify `$updateParentNodeInfo->recursive()` here?

Reply:
That was also one of my first ideas, but the parent is the outer collection with even more nodes not related to what we are changing.

Reply:
The AbstractCopy adds another NodeInfo